x-pack/plugins/transform/public/app/sections/create_transform/components/aggregation_list/sub_aggs_section.tsx

```diff
@@ -54,11 +57,49 @@ export const SubAggsSection: FC<{ item: PivotAggsConfig }> = ({ item }) => {
     }
     return nestingLevel <= MAX_NESTING_SUB_AGGS;
   }, [item]);
+  const { handleFieldStatsButtonClick, populatedFields } = useFieldStatsTrigger();
 
+  const options = useMemo(() => {
+    const opts: EuiComboBoxOptionOption[] = [];
+    state.aggOptions.forEach(({ label, field, options: aggOptions }: DropDownOptionWithField) => {
+      const isEmpty = populatedFields && field.id ? !populatedFields.has(field.id) : false;
+
+      const aggOption: DropDownOption = {
+        isGroupLabel: true,
+        key: field.id,
+        searchableLabel: label,
+        // @ts-ignore Purposefully passing label as element instead of string
+        // for more robust rendering
+        label: (
+          <FieldStatsInfoButton
+            isEmpty={populatedFields && !populatedFields.has(field.id)}
+            field={field}
+            label={label}
+            onButtonClick={handleFieldStatsButtonClick}
+          />
+        ),
+      };
+
+      if (aggOptions.length) {
+        opts.push(aggOption);
+        opts.push(
+          ...aggOptions.map((o) => ({
+            ...o,
+            isEmpty,
+            label: o.label,
```

Comment:
`o` is spread here, so `label: o.label` shouldn't be needed

Reply:
Updated here [`9b9ee5b` (#195485)](https://github.com/elastic/kibana/pull/195485/commits/9b9ee5b252977641fe1fa4b9550acbb855c48eed)